src/main/java/com/rusticisoftware/cloud/v2/client/api/DispatchApi.java

```diff
@@ -128,8 +128,8 @@ public void createDispatches(CreateDispatchListSchema dispatchList) throws ApiEx
     apiClient.invokeAPI(localVarPath, "POST", localVarQueryParams, localVarPostBody, localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames, null);
   }
   /**
-   * DestinationsByTenantAndDestinationId
-   * Delete the destination with destinationId &#x3D; {destinationId}.
+   * Delete destination 
+   * Delete the destination with &#x60;destinationId&#x60; 
```

Comment:
Yeah, I think the codegen may have done some weirdness when generating the javadoc stuff. That's probably another issue then.